stm32cube/stm32f7xx/drivers/include/stm32f7xx_hal_cryp.h

```diff
@@ -330,6 +330,17 @@ typedef  void (*pCRYP_CallbackTypeDef)(CRYP_HandleTypeDef *hcryp);    /*!< point
 #define CRYP_KEYIVCONFIG_ALWAYS        0x00000000U            /*!< Peripheral Key and IV configuration to do systematically */
 #define CRYP_KEYIVCONFIG_ONCE          0x00000001U            /*!< Peripheral Key and IV configuration to do only once      */
 
+/**
+  * @}
+  */
+
+/** @defgroup CRYP_Configuration_Skip CRYP Key and IV Configuration Skip Mode
+  * @{
+  */
+
+#define CRYP_KEYIVCONFIG_ALWAYS        0x00000000U            /*!< Peripheral Key and IV configuration to do systematically */
```

Comment:
This block looks wrong, since these definitions (and doxygen doc) are already there. This will lead to duplicated definitions of `CRYP_KEYIVCONFIG_ALWAYS` and `CRYP_KEYIVCONFIG_ONCE`. It looks like this is also the case for some other (following) changes in this pull request. Is this supposed to be duplicated?

Reply:
Ok, yes some duplicates appear. Removing